src/stashcache.py

```diff
@@ -314,10 +315,12 @@ def generate_cache_scitokens(vo_data: VOsData, resource_groups: List[ResourceGro
                     continue
                 issuer_blocks.append(_get_scitokens_issuer_block(vo_name, authz['SciTokens'],
                                                                  dirname, suppress_errors))
+                allowed_vos.append(vo_name)
 
     issuer_blocks_str = "\n".join(issuer_blocks)
+    allowed_vos_str = ", ".join(allowed_vos)
 
-    return template.format(**locals()).rstrip() + "\n"
+    return template.format(issuer_blocks_str=issuer_blocks_str, allowed_vos=allowed_vos_str).rstrip() + "\n"
```

Comment:
```suggestion
    return template.format(**locals()).rstrip() + "\n"
```
I don't get the point of changing the code style here, especially since it creates an inconsistency between issuer_blocks_str and allowed_vos_str.

Reply:
I just thought it was scary to put all locals in the template.  But it should be fine since we control the template.